tests/t5/test_modeling_tf_t5.py

```diff
@@ -454,6 +476,27 @@ def test_train_pipeline_custom_model(self):
 @require_sentencepiece
 @require_tokenizers
 class TFT5GenerationIntegrationTests(unittest.TestCase):
+    @slow
+    def test_greedy_xla_generate_simple(self):
+        model = TFT5ForConditionalGeneration.from_pretrained("t5-small", min_length=-1)
+        tokenizer = T5Tokenizer.from_pretrained("t5-small")
+
+        sentence = "Translate English to German: Today is a beautiful day."
+        input_ids = tokenizer(sentence, return_tensors="tf", padding=True).input_ids
+
+        xla_generate = tf.function(model.generate, jit_compile=True)
```

Comment:
This is the only line one has to add now to make it xla_compile-able - works with T5 and GPT2 now. Ok for you @Rocketknight1 @gante ?